tests/foreman/api/test_discoveryrule.py

```diff
@@ -171,8 +171,11 @@ def test_positive_multi_provision_with_rule_limit(
 
     :CaseImportance: High
     """
+
     discovered_host1 = module_target_sat.api_factory.create_discovered_host()
+    request.addfinalizer(module_target_sat.api.Host(id=discovered_host1['id']).delete)
     discovered_host2 = module_target_sat.api_factory.create_discovered_host()
+    request.addfinalizer(module_target_sat.api.Host(id=discovered_host2['id']).delete)
```

Comment:
```suggestion
    request.addfinalizer(module_target_sat.api.DiscoveredHost(id=discovered_host2['id']).delete)
```

Reply:
Are these not identical actions?

Reply:
no, when we provision discovered_host with rule in next step, discovered host will be moved to host entity